05_iigo/index.tex

```diff
@@ -477,6 +477,40 @@ \subsection{End of Session Actions}
     \item The Judge has the \emph{power} to decide to hold an election for a new President.
 \end{enumerate}
 
+\section{Executive branch implementation}
+
+The executive branch has been implemented to facilitate the use of powers given to the president. Because any agent could potentially become the president, implementation of the executive branch has been split into two interconnected parts:
+\begin{itemize}
+\item Client-side implementation of presidential functions, which map directly to the powers given to the president.
+\item Server-side implementation of executive branch actions, which calls the presidential functions only on the currently residing president.
+\end{itemize}
```

Comment:
I like this a lot. Maybe we should move this to the overview of infra because it applies to most powers held by roles